spec.md

```diff
@@ -386,6 +388,8 @@ It is therefore RECOMMENDED that implementations avoid such strings.
 | tags | array of strings | Tags provide a flexible mechanism to expose a classification, attribute, or base technology of a service, enabling equivalent services to be swapped out without changes to dependent logic in applications, buildpacks, or other services. E.g. mysql, relational, redis, key-value, caching, messaging, amqp. |
 | requires | array of strings | A list of permissions that the user would have to give the service, if they provision it. The only permissions currently supported are `syslog_drain`, `route_forwarding` and `volume_mount`. |
 | bindable* | boolean | Specifies whether Service Instances of the service can be bound to applications. This specifies the default for all plans of this service. Plans can override this field (see [Plan Object](#plan-object)). |
+| instances_retrievable | boolean | Specifies whether the [Fetching a Service Instance](#fetching-a-service-instance) endpoint is supported for all plans. |
+| bindings_retrievable | boolean | Specifies whether the [Fetching a Service Binding](#fetching-a-service-binding) endpoint is supported for all plans. |
```

Comment:
For backward compatibility the default should be `false`. State it explicitly.
For both  `instances_retrievable` and `bindings_retrievable`.

Reply:
@dotchev We cannot state that new fields are required in the catalog. Platforms should treat the absence of the field the same as it would treat the presence of the field set to false.